dealroom_firestore_connector/__init__.py

```diff
@@ -458,6 +482,12 @@ def set_history_doc_refs(
     else:
         count_history_refs = 0
 
+    document_matches_by_final_url = key_found == "final_url"
+    if document_matches_by_final_url:
```

Comment:
why was this removed? `if count_history_refs:` @GuillermoDR

Reply:
It's the same thing, `count_history_refs` will be initialised if `document_matches_by_final_url` has a value